docs/tutorials/framedata.md

```diff
@@ -0,0 +1,102 @@
+# Accessing Data from the Video Source
+
+This tutorial will provide an example of accessing data from a video source during acquisition. 
+
+## Configure `Runtime`
+
+To start, we'll create a `Runtime` object and configure the streaming process.
+
+```python
+import acquire
+
+# Initialize a Runtime object
+runtime = acquire.Runtime()
+
+# Initialize the device manager
+dm = runtime.device_manager()
+
+# Grab the current configuration
+config = runtime.get_configuration() 
+
+# Select the radial sine simulated camera as the video source
+config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sin") 
+
+# Set the storage to trash to avoid saving the data
+config.video[0].storage.identifier = dm.select(acquire.DeviceKind.Storage, "Trash")
+
+# Set the time for collecting data for a each frame
+config.video[0].camera.settings.exposure_time_us = 5e4  # 50 ms
+
+config.video[0].camera.settings.shape = (1024, 768)
+
+# Set the max frame count to 2**(64-1) the largest number supported by Uint64 for essentially infinite acquisition
+config.video[0].max_frame_count = 100 # collect 100 frames
+
+# Update the configuration with the chosen parameters 
+config = runtime.set_configuration(config) 
+```
+## Working with `AvailableData` objects
+
+During Acquisition, the `AvailableData` object is the streaming interface, and this class has an `frames` method which iterates over the `VideoFrame` objects in `AvailableData`. Once we start acquisition, we'll utilize this iterator method to list the frames. 
+
+```python
+# To increase the likelihood of `AvailableData` containing data, we'll utilize the time python package to introduce a delay before we create our `Availabledata` object
+import time
+
+# start acquisition
+runtime.start()
+
+# time delay of 0.5 seconds
+time.sleep(0.5)
+
+# grab the packet of data available on disk for video stream 0. This is an AvailableData object.
+available_data = runtime.get_available_data(0) 
+```
+Once `get_available_data` is called the `Availabledata` object will be locked into memory, so the camera will timeout if `Availabledata`t isn’t released once it returns to that part of the RAM that contains `Availabledata`.
+
+There may not be data available, in which case our variable `available_data` would be a NoneType. To avoid errors associated with this circumstance, we'll only grab data if `available_data` is not a NoneType.

```

Comment:
```suggestion
There may not be data available, in which case our variable `available_data` would be `None`. To avoid errors associated with this circumstance, we'll only grab data if `available_data` is not `None`.

```